viewer/Viewer.h

```diff
@@ -111,23 +124,58 @@ namespace Enki
 		typedef QMapIterator<const std::type_info*, const std::type_info*> ManagedObjectsAliasesMapIterator;
 		ManagedObjectsAliasesMap managedObjectsAliases;
 		
+		typedef std::pair<QString, unsigned int> viewerMessage;
+		std::list<viewerMessage> messageList;
+		QString controlError1;	// dislpayed if user try to move an object in trackball mode : action forbiden to prevent glitchies
+		QString controlError2;	// dislpayed if user try to move camera in trackball mode
+		QString controlHelp;	// dislpayed if user press F1
+
+		struct ExtendedAttributes
+		{
+			bool movableByPicking;
+
+			ExtendedAttributes():movableByPicking(false){};
+		};
+		std::map<PhysicalObject*, ExtendedAttributes> objectExtendedAttributesList;
```

Comment:
I see the rationale. The problems are:

1. If an object is removed from the world, it is not removed from this map. You could say that it's not a drama but it might leak some objects.
2. Because of one, the design of Enki is that the core engine handles all informations, through UserData. Actually this question is a bit tricky as we discussed, and show be solved more globally through an entity-component system.

So for new I'm ok with that approach, but I would like that we add a note that it is the responsibility of the user of the extended attribute to remove them when the object is removed, and we should add a function to clear an object from this list.